cmd/tsbs_generate_data/main.go

```diff
@@ -174,14 +178,31 @@ func main() {
 	}
 
 	rand.Seed(seed)
-	out := bufio.NewWriterSize(os.Stdout, inputBufSize)
+
+	// Prepare output file/STDOUT
+	var out  *bufio.Writer
+	var file io.Writer
+	var err  error
+	if len(fileName) > 0 {
+		// Write output to file
+		file, err = os.Create(fileName)
+		if err != nil {
+			panic("cannot open file " + fileName)
```

Comment:
We are using `fatal()` here so it can 1) be tested and 2) has a prettier output than panic. Also it would be helpful if the error message included the err, so maybe `fatal("cannot open file %s: %v", fileName, err)`

Reply:
Let's completely rewrite this section. I'll extract in to separate function like GetBufferedWriter() as an analogue to GetBufferedReader(). Later we can extract those functions one layer higher into `common`  or `util`

Reply:
Good idea -- I do have plans to refactor the two generate binaries into a common lib like `load` and `query`